src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -108,8 +109,7 @@ public static void BasicAuthenticationEmptyUser_throw_ArgumentException()
         Assert.True(exception.Message.ToLower().Contains(paraMessage), string.Format("Expected exception message to contain: '{0}', actual: '{1}'", paraMessage, exception.Message));
     }
 
-    [Fact]
-    [ActiveIssue(270)]
+    [ConditionalFact(nameof(Domain_Joined_Tests_Enabled))]
```

Comment:
Reviewers: we have the green light to use C# 6 syntax.  CoreFx has already migrated or is migrating their [ConditionalFact] attributes to use the new "nameof".  This not only gives compile-time checking, but VS's "Find all references" works well to find every test using this condition.